packages/python-certifi/python-certifi.spec

```diff
@@ -5,19 +5,17 @@
 %global pypi_name certifi
 
 Name:           %{?scl_prefix}python-%{pypi_name}
-Version:        2020.6.20
-Release:        2%{?dist}
+Version:        2021.10.8
+Release:        1%{?dist}
 Summary:        Python package for providing Mozilla's CA Bundle
 
 License:        MPL-2.0
 URL:            https://certifiio.readthedocs.io/en/latest/
 Source0:        https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
-Patch0:         certifi-2020.6.20-use-system-cert.patch
```

Comment:
please don't drop this patch (or the `ca-certificates` requires), we want to use the system certs, not the bundled ones.

(in reality, you don't need to bump certifi because of this, as its content never changes for us)